### PR TITLE
Bug fix for negative probabilities

### PR DESCRIPTION
**Pull request by nfwvogt**

### CODE REVIEW

File: .github/workflows/ci_tests.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   # push:
   #   branches: [main]
   pull_request:
-    branches: [main]
+    branches: [main, "dev0.8.1"]
```
Comment (kbarkhqs) [inline .github/workflows/ci_tests.yml:7]: The "dev0.8.1" part can be removed

File: .github/workflows/ci_tests.yml [hunk 2]
```diff
@@ -116,42 +112,42 @@ jobs:
...
+  #       with:
+  #         python-version: ${{ matrix.python.py }}
+  #     - name: linux wheels
+  #       uses: messense/maturin-action@v1
+  #       with:
+  #         maturin-version: latest
+  #         command: build
+  #         manylinux: 2014
+  #         args: -i python${{ matrix.python.py }} --out wheels -m qoqo-quest/Cargo.toml --locked
+  #     - name: test install
```
Comment (kbarkhqs) [inline .github/workflows/ci_tests.yml:148]: Is this intentional?

File: .github/workflows/python_ci_tests.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ on:
   # push:
   #   branches: [main]
   pull_request:
-    branches: [main]
+    branches: [main, "dev0.8.1"]
```
Comment (kbarkhqs) [inline .github/workflows/python_ci_tests.yml:7]: Similarly, dev0.8.1 can be removed here

**Review by kbarkhqs — APPROVED** [review]

File: .github/workflows/ci_tests.yml (hunk 2 above)
Comment (nfwvogt) [reply .github/workflows/ci_tests.yml:148]: Yes, we have those test somewhere else

File: .github/workflows/python_ci_tests.yml (hunk 3 above)
Comment (nfwvogt) [reply .github/workflows/python_ci_tests.yml:7]: Not at the moment or the CI won't run as we are not merging into main

**Pull request merged**